keyboards/lets_split/rules.mk

```diff
@@ -1,11 +1,4 @@
-SRC += matrix.c \
-	   i2c.c \
-	   split_util.c \
-	   serial.c \
-	   ssd1306.c
```

Comment:
Could you add `SRC += ssd1306.c` to the OLED_test keymap's rules.mk? 

That should fix it's error.

Reply:
Done, builds locally but produces a `The firmware is too large!` error. Given the name of the keymap, OLED_sample, would it be worth disabling some features to reduce the size?

Reply:
Yeah. Start with mousekeys, though.

Reply:
All good with just the mouse keys disabled.